share/doc/xml/dax.xml

```diff
@@ -129,6 +129,18 @@ unix% ds9 -analysis $ASCDS_CONTRIB/config/ciao.ds9 ...
 
 </DESC>
 
+
+<ADESC title="Changes in the 4.13.1 (January 2021) release">
+  <PARA>
+    Added additional redshifted absorption spectral models.
+    Added energy limits to the photon and energy flux calculations
```

Comment:
As mentioned in  https://github.com/cxcsds/ciao-contrib/issues/443#issuecomment-784319400 I think we should say that previously the flux was calculated over the full energy range of the instrument (so ~0.3-11 keV I think, but can't guarantee, so maybe we shouldn't give explicit values?)